network/hermes/create-conn.sh

```diff
@@ -8,5 +8,14 @@ set -e
 echo "Initiating connection handshake..."
 $HERMES_BINARY --config $CONFIG_DIR create connection --a-chain test-1 --b-chain test-2
 
-sleep 2
-$HERMES_BINARY --config $CONFIG_DIR create channel --a-chain test-1 --a-connection connection-0 --a-port transfer --b-port transfer
+
+
+# a short living client is needed for client update proposal test
+echo "Initiating short living client..."
```

Comment:
Is it possible to move it into the integration tests?
Cause this seems unnecessary in Neutron itself